README.md

```diff
@@ -18,18 +18,24 @@ Clone the repository in your computer:
 $ git clone git@github.com:adriacidre/ali.git
 ```
 
-Then set up your ./basrc or ./zshrc with some commands like the ones below with your own $GOPATH
+Then set up your ./zshrc with the following command:
+
+```
+$ make install
+```
+
+And set up the ali with some commands like the ones below with your own $GOPATH
```

Comment:
This sentence does not make any sense 😹 Let's remove it and automate the alias creation.

Reply:
Ok